spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/Benchmark.java

```diff
@@ -91,7 +91,7 @@ public Benchmark(
         this.paths = paths;
         this.spark = spark;
         this.dataFileSystem = dataFileSystem;
-        this.sqlQuerySupplier = Suppliers.memoize(() -> buildSqlQueries(spark));
+        this.sqlQuerySupplier = Suppliers.memoize(() -> buildSqlQueries(spark.get()));
```

Comment:
the returned list of `SqlQuery` objects close over the spark session returned by first invocation `spark.get()`,  so the "live reload" won't work, I think... need to plumb the supplier logic all the way through the SqlQuery logic (where it's used, ironically, to create a `Supplier<Dataset<Row>>` so should be doable)